src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1154,4 +1154,7 @@ This version used in: {2}</value>
   <data name="Add_argument_name_0" xml:space="preserve">
     <value>Add argument name '{0}'</value>
   </data>
+  <data name="Convert_if_to_switch" xml:space="preserve">
+    <value>Convert if to switch</value>
```

Comment:
This should be probably specialized for C# and VB. I don't have any idea how should it be done. Perhaps a field is ok?